AutoPartsShop.WebUI/Pages/FetchData.razor

```diff
@@ -0,0 +1,57 @@
+﻿@page "/fetchdata"
+@inject HttpClient Http
+
+<PageTitle>Weather forecast</PageTitle>
```

Comment:
The code here seems to be useless as well. It's about weather and I don't see it use anywhere in your app.